ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -2152,6 +2152,16 @@ router_general:
 router_bgp:
   as: < bgp_as >
   router_id: < IPv4_address >
+  distance:
+    external_routes: < 1-255 >
+    internal_routes: < 1-255 >
+    local_routes: < 1-255 >
+  maximum_paths:
+    ec_paths: < 1-600 >
+    ecmp_routes: < 1-600 >
+  updates:
```

Comment:
Hi Claus, unfortunately "update" is a reserved keyword.. so can't be used as key (if there is no workaround for that). 

For the paths it is possible to switch to singular.

Reply:
Claus, I've just doublechecked. EOS CLI also says "maximum_path**s**". How about changing to this:

```
maximum_paths:
  ec: < 1-600 >
  ecmp: < 1-600 >
updates:
```

Regarding the key "updates", I'm still unsure. Can it remain plural, as "update" is reserved keyword, or should we rather pick a completely different term?

Reply:
I have a hack. We can use ```update``` knob but while retrieving data we should not use ```.``` method (like router_bgp.update) but retrieve data by router_bgp['update']. I know this against our rules but this works and we avoid plural form.

Reply:
Since this will cause all kinds of issues in templates or inline jinja, we should just keep the "updates" as it is now.

Reply:
@tgodaA yup, using bracket notation was mentioned in the Ansible doc as well. Thinks its better to adjust the key though, otherwise there is bracket notation just for this case and nobody will remember why in like 3 months :-D